ansible/idr-playbooks/group_vars/omero-hosts.yml

```diff
@@ -20,5 +21,58 @@ omero_web_public_password: "{{ idr_secret_omero_web_public_password | default('p
 
 omero_prestart_file: "{{ playbook_dir }}/files/IDR-OMERO-52-omero.j2"
 
-omero_logmonitor_slack_token: "{{ idr_secret_omero_logmonitor_slack_token | default(None) }}"
-omero_logmonitor_slack_channel: "#logs"
+
+######################################################################
+# mapr
+
+omero_web_app_packages:
+#  - "omero-mapr==0.1.1"
+  - "git+git://github.com/aleksandra-tarkowska/omero-mapr.git@0.1.6#egg=omero_mapr"
```

Comment:
It'll be in a later commit/PR https://github.com/manics/infrastructure/commit/4668d16b3158fcc80fde9209fb9729689b1b18f8
https://github.com/manics/infrastructure/commit/4c11c5cf6f37e40fa44ba452c637f15428b6e064